---
source: crates/runtime/tests/fn_call_default.rs
expression: snapshot
---
# Input:
fn f(a=10):
  print a

f(1,2)


# Result (error):
In test_func at 0..0
Error: expected at most 1 args, got 2


# Stdout:

